### ISSUE TITLE
Accuracy for coding/intron only when in a transcript

### ISSUE THREAD

**Issue by soi** [weberlab-hhu/GeenuFF#56]

**Comment by soi** [comment]
Wrote a script that outputs the accs given a prediction file. It seemed like it could not be done in keras.
